src/test/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingtoolsapi/integration/ReportDefinitionIntegrationTest.kt

```diff
@@ -455,7 +454,6 @@ class ReportDefinitionIntegrationTest : IntegrationTestBase() {
 
     assertThat(field.defaultsort).isFalse()
     assertThat(field.filter).isNotNull()
-    assertThat(field.filter!!.dynamicOptions!!.returnAsStaticOptions).isTrue()
```

Comment:
Was this assertion not needed?

Reply:
I removed the dynamic option properties that the front-end didn't need to know about: src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportinglib/controller/model/DynamicFilterOption.kt

Reply:
I see, it's the controller model so not returned there.